packages/atomic/src/components/facets/facet-show-more-less/facet-show-more-less.tsx

```diff
@@ -11,6 +11,8 @@ interface FacetShowMoreProps {
   canShowMoreValues: boolean;
   onShowMore(): void;
   onShowLess(): void;
+  showMoreRef?: (element?: HTMLButtonElement) => void;
+  showLessRef?: (element?: HTMLButtonElement) => void;
```

Comment:
We don't need `showLessRef`, but I think it would be weird to have only one of the two props.